patches/extra/ungoogled-chromium/remove-uneeded-ui.patch

```diff
@@ -13,21 +13,28 @@
 
 --- a/chrome/browser/resources/history/side_bar.html
 +++ b/chrome/browser/resources/history/side_bar.html
-@@ -101,11 +101,6 @@
-         $i18n{historyMenuItem}
-         <paper-ripple></paper-ripple>
-       </a>
--      <a href="/syncedTabs" class="page-item" path="syncedTabs"
--          on-click="onItemClick_">
--        $i18n{openTabsMenuItem}
--        <paper-ripple></paper-ripple>
--      </a>
-       <div class="separator"></div>
-       <a id="clear-browsing-data"
-           href="chrome://settings/clearBrowserData"
+@@ -104,11 +104,6 @@
+           $i18n{historyMenuItem}
+           <paper-ripple></paper-ripple>
+         </a>
+-        <a role="menuitem" href="/syncedTabs" class="page-item"
+-            path="syncedTabs" on-click="onItemClick_">
+-          $i18n{openTabsMenuItem}
+-          <paper-ripple></paper-ripple>
+-        </a>
+         <div class="separator"></div>
+         <a role="menuitem" id="clear-browsing-data"
+             href="chrome://settings/clearBrowserData"
+@@ -131,4 +126,4 @@
+         <iron-icon icon="cr:info-outline"></iron-icon>
+         <div>$i18nRaw{sidebarFooter}</div>
+       </div>
+-    </div>
+\ No newline at end of file
++    </div>
```

Comment:
The above three lines just look weird to me. I don't understand why quilt is even touching this portion of the file, let alone adding `\ No newline at the end of file` *before* the end of the file... This looks like a quilt bug to me, but perhaps this is just intended behavior that I am not aware of.